apps/arweave/src/ar_http.erl

```diff
@@ -179,6 +181,7 @@ handle_info({gun_error, PID, Reason},
 					prometheus_gauge:dec(outbound_connections),
 					ok
 			end,
+			ar_peers:inactivate_peer(Peer),
```

Comment:
We should be getting here on a regular connection shutdown (occurs after a [minute](https://github.com/ArweaveTeam/arweave/blob/master/apps/arweave/src/ar_http.erl#L249)) too.